packs/cloudflare/tests/test_action_get_ips.py

```diff
@@ -0,0 +1,128 @@
+import yaml
+import requests
+import requests_mock
+
+from st2tests.base import BaseActionTestCase
+
+from get_ips import GetIPs
+
+__all__ = [
+    'GetIPsTestCase'
+]
+
+MOCK_CONFIG_BLANK = ""
+
+MOCK_CONFIG_NO_SCHEMA = """
+api_host: "api.cloudflare.com"
+"""
+
+MOCK_CONFIG_FULL = """
+api_host: "mock://api.cloudflare.com"
+"""
+
+MOCK_DATA_INVALID_JSON = "{'dd': doo}"
+
+MOCK_DATA_SUCCESS = """
```

Comment:
Minor thing - I usually prefer to put raw response fixtures like that in a file (e.g. `packs/cloudflare/tests/fixture/list_ips_success.json` or similar) and load file content in the code.

This way it's a bit easier to update those files and make sure they indeed contain valid JSON (e.g. we can add a CI step which runs syntax check on all the *.json files or similar).

If needed, we can also add some helper functions to the base test class which allows you to easily load content from a file (although, even without that function it's only 1-3 lines of Python, but the function could at least enforce a convention for fixture file locations - e.g. `packs/<pack name>/tests/fixtures/<blah>/<fixture file name>`).